src/Workspaces/VisualBasic/Portable/FindSymbols/VisualBasicReferenceFinder.vb

```diff
@@ -0,0 +1,46 @@
+﻿Imports System.Composition
+Imports System.Threading
+Imports System.Threading.Tasks
+Imports Microsoft.CodeAnalysis.FindSymbols.Finders
+Imports Microsoft.CodeAnalysis.Host.Mef
+
+Namespace Microsoft.CodeAnalysis.FindSymbols
+    <ExportLanguageService(GetType(ILanguageServiceReferenceFinder), LanguageNames.VisualBasic), [Shared]>
+    Friend Class VisualBasicReferenceFinder
+        Implements ILanguageServiceReferenceFinder
+
+        Public Async Function DetermineCascadedSymbolsAsync(namedType As INamedTypeSymbol, project As Project, cancellationToken As CancellationToken) As Task(Of IEnumerable(Of ISymbol)) Implements ILanguageServiceReferenceFinder.DetermineCascadedSymbolsAsync
+            If namedType.Language = LanguageNames.VisualBasic AndAlso project.Language = LanguageNames.VisualBasic Then
+                Dim compilation = Await project.GetCompilationAsync(cancellationToken).ConfigureAwait(False)
+
+                ' If this is a WinForms project, then the VB 'my' feature may have synthesized 
+                ' a property that would return an instance of the main Form type for the project.
+                ' Search for such properties and cascade to them as well.
+                Dim myPropertySymbols = GetMatchingMyPropertySymbols(namedType, DirectCast(compilation, VisualBasicCompilation), cancellationToken).Distinct().ToList()
+                If myPropertySymbols.Count > 0 Then
```

Comment:
And even so, if you've already constructed an empty list, does it mean anything different to return null?